.circleci/config.yml

```diff
@@ -6,36 +6,28 @@ orbs:
 executors:
   circleci_machine:
     docker:
-      - image: circleci/node:14.15
+      - image: cimg/node:14.15
     working_directory: ~/repo
 
 commands:
   setup_python_env:
     description: Setup Python environment
     steps:
       - run:
-          name: Install python3
+          name: Install python3.8
           command: |
+            sudo apt-get install software-properties-common
+            sudo add-apt-repository ppa:deadsnakes/ppa -y
             sudo apt update
-            sudo apt install python3 python3-pip
+            sudo apt install python3.8 python3-pip python3-venv
       - run:
           name: Install Python environment
           command: |
-            # Get rid of pyenv stuff
-            sudo rm -rf .pyenv/ /opt/circleci/.pyenv/
-            # Download and install miniconda
-            curl https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh  > miniconda.sh
-            bash miniconda.sh -b -p $HOME/miniconda
-            # Create an anaconda virtualenv for python ${CONDA_ENV_PY_VERSION} and make that the default python interpreter
-            echo ". /home/circleci/miniconda/etc/profile.d/conda.sh" >> $BASH_ENV
-            echo "conda deactivate; conda activate kedro-viz" >> $BASH_ENV
-            . /home/circleci/miniconda/etc/profile.d/conda.sh
-            conda create --name kedro-viz python=${CONDA_ENV_PY_VERSION} -y
-            source $BASH_ENV
```

Comment:
@limdauto this is more for my curiosity - I wonder why won't we need conda as the default python interpreter with the new setup? Is it becuase of the switch to use `cimg`?